packages/mongodb-chatbot-server/src/processors/makeDefaultFindVerifiedAnswer.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  Embedder,
+  FindNearestNeighborsOptions,
+  VerifiedAnswerStore,
+} from "mongodb-rag-core";
+import { FindVerifiedAnswerFunc } from "./FindVerifiedAnswerFunc";
+
+export type MakeDefaultFindVerifiedAnswerFuncArgs = {
+  embedder: Embedder;
+  store: VerifiedAnswerStore;
+  findNearestNeighborsOptions?: Partial<FindNearestNeighborsOptions>;
+};
+
+/**
+  Basic implementation of FindVerifiedAnswerFunc.
+ */
+export const makeDefaultFindVerifiedAnswer = ({
+  embedder,
+  store,
+  findNearestNeighborsOptions,
+}: MakeDefaultFindVerifiedAnswerFuncArgs): FindVerifiedAnswerFunc => {
+  return async ({ query }) => {
+    const { embedding: queryEmbedding } = await embedder.embed({
+      text: query,
+    });
+
+    const answers = await store.findNearestNeighbors(
+      queryEmbedding,
+      findNearestNeighborsOptions
+    );
+
+    // Sort by score descending
+    answers.sort((a, b) => b.score - a.score);
```

Comment:
would it not already be sorted?

Reply:
I wasn't sure so I added it here, I should probably make it a guarantee of the API?

Reply:
Here's my hangup: just because we say something like this in a doc comment:
```ts
  /**
    Find nearest neighbors to the given vector.

    Return array must be ordered by score, descending.
   */
  findNearestNeighbors(...)
```
...doesn't mean that implementers will actually do it.